src/indra_cogex/client/queries.py

```diff
@@ -778,7 +781,7 @@ def get_evidences_for_mesh(
 
     query_params = {}
     if child_terms:
-        match_terms = {norm_mesh} | child_terms
+        match_terms = list({norm_mesh} | child_terms)
```

Comment:
I'm not sure this change is needed, since we use list below at line 787 in `query_params["mesh_terms"] = list(match_terms)`

Reply:
Reverted back to original logic in: d7b4321c